### ISSUE TITLE
[PROPOSAL] We urgently need to remove the panic of real players due to random bans

### ISSUE THREAD

**Issue by IvanSkazkin** [sunflower-land/sunflower-land#614]
**Hello.**
I see bans happening now.
I sit in many chats and ban not only multi-accounts, but also ordinary players.
**In this regard, ordinary players are very scared for their assets.**
_Perhaps we'd better come up with some way to reduce the panic among real players._
For example, do not ban the farm immediately. And put an unlock account for 10 wheat
This would encourage people to level up (and give you time to better set up the anti-bot system).
Give the account 3 warnings, after which it will be banned. (Increase the cost of unlocking wheat, after each ban by 10 wheat + 1 gold)
It is important that wheat that was grown before or bought on the market does not count. The player must spend time growing himself.
During the freeze, prohibit people from transferring any vegetables and accepting any resources other than NFTs. Allow the creation of NFTs and transfer them to opensea.
There are chats where many people say that they are sitting without a bot, but they got banned. I understand that you need to ban multi-accounts, but you need to somehow secure ordinary players
I have 1 account and have been playing since April 8th. Never used a bot. I played only on PC and on the phone at the university. And my farm is under review. I bought a carrot sword from the opensea market for $130 and now I'm scared that I might lose money, so I put all my NFTs on the market. Many of the chat do the same. They simply take NFTs out of the game because they are afraid of losing them.

_Regular players complain that they don't get NFT hare because of the antibot_

**Comment by arthurbolsoni** [comment]
I don't even keep a nft in the Land for fear of it being listed in the "guess is is bot" from the server code that is not available to the community making the project totally dependent and not decentralized as a blockchain proposal. I don't have any control of one of my nft at the moment I put it in Land.

**Comment by IvanSkazkin** [comment]
> Я даже не держу nft в земле из-за боязни, что он будет указан в списке «предполагаю, что это бот» из кода сервера, который недоступен сообществу, что делает проект полностью зависимым и не децентрализованным, как предложение блокчейна. Я не имею никакого контроля над своим nft в тот момент, когда я помещаю его в землю.

I agree, the whole point is lost in the detracilized finger

**Comment by HarshalKudale** [comment]
They should give access to inventory contract to farm owners.  Where farms owners should be able to withdraw only nfts from the  inventory. Crops should not be allowed to be taken out for blacklisted accounts. This goes for temp/wrongfully banned accounts.

For permanant banned accounts their assets should be auctioned to give them some lesson.

**Comment by IvanSkazkin** [comment]
Now I'm even afraid to plant sunflowers quickly. I'm afraid they'll think I'm a bot. And I really already started doing it quickly after 10.000 planted sunflowers)

**Comment by simiyet** [comment]
Unfortunately I am one of those. I did not use bot, did not use any automation programs, do not multiple accounts. All I did was a p2p trade on 15th but back then trading was allowed on discord. I honestly think I was flagged by system because of this. I have 4 NFTs and alot of resources on my farm atm and I can not withdraw them now.

Now I am scared, genuinely. Will I not be able trade with people now? What is gonna happen if I trade with someone else again outside of opensea?
